mkdocs.yml

```diff
@@ -0,0 +1,6 @@
+site_name: pod-image-swap-webhook
```

Comment:
This needs to be changed

Reply:
Fixed. I copied it from there as a template, but probably missed to change that.